resources/views/layouts/default.blade.php

```diff
@@ -513,7 +513,7 @@
             @endcan
 
             @can('backend.interact')
-                <li>
+                <li class="treeview">
```

Comment:
What does this actually do?

Reply:
this fixes the rounded corner on the settings menu

Reply:
this is also used on the Assets and Reports menus

Reply:
Ahh, right! Thanks!